src/EnergyPlus/DaylightingManager.cc

```diff
@@ -1360,58 +1362,54 @@ void CalcDayltgCoeffsMapPoints(EnergyPlusData &state, int const ZoneNum)
                 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylIllFacSun(state.dataGlobal->HourOfDay,
                                                                           {1, MaxSlatAngs + 1},
-                                                                          {1, DataDaylighting::MaxRefPoints},
+                                                                          {1, numRefPts},
                                                                           {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) =
                 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylIllFacSunDisk(
                 state.dataGlobal->HourOfDay,
                 {1, MaxSlatAngs + 1},
-                {1, DataDaylighting::MaxRefPoints},
+                {1, numRefPts},
                 {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) = 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylSourceFacSun(
                 state.dataGlobal->HourOfDay,
                 {1, MaxSlatAngs + 1},
-                {1, DataDaylighting::MaxRefPoints},
+                {1, numRefPts},
                 {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) = 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylSourceFacSunDisk(
                 state.dataGlobal->HourOfDay,
                 {1, MaxSlatAngs + 1},
-                {1, DataDaylighting::MaxRefPoints},
+                {1, numRefPts},
                 {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) = 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylBackFacSun(state.dataGlobal->HourOfDay,
                                                                            {1, MaxSlatAngs + 1},
-                                                                           {1, DataDaylighting::MaxRefPoints},
+                                                                           {1, numRefPts},
                                                                            {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) =
                 0.0;
             state.dataDaylightingData->IllumMapCalc(MapNum).DaylBackFacSunDisk(
                 state.dataGlobal->HourOfDay,
                 {1, MaxSlatAngs + 1},
-                {1, DataDaylighting::MaxRefPoints},
+                {1, numRefPts},
                 {1, state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins}) = 0.0;
         }
-        NRF = state.dataDaylightingData->IllumMapCalc(MapNum).TotalMapRefPoints;
-
-        // MapWindowSolidAngAtRefPt.allocate( NRF, ZoneDaylight( ZoneNum ).NumOfDayltgExtWins ); //Inactive
-        // MapWindowSolidAngAtRefPtWtd.allocate( NRF, ZoneDaylight( ZoneNum ).NumOfDayltgExtWins ); // Not an array anymore
 
-        for (IL = 1; IL <= NRF; ++IL) {
```

Comment:
Got it, NRF was already a dynamic variable for the amount of reference points, but now it's just not needed since we already have `numRefPts`.  :+1: